packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -129,22 +129,26 @@ export class RedirectsMiddleware extends MiddlewareBase {
       /** return Response redirect with http code of redirect type **/
       switch (existsRedirect.redirectType) {
         case REDIRECT_TYPE_301:
-          return NextResponse.redirect(redirectUrl, 301);
+          return NextResponse.redirect(redirectUrl, {
```

Comment:
@matkovskyi could you please take a look here? Would this be in line with what you expect of SXA rewrites?
I'm talking mainly about the statusText and headers (which might be set in middleware and preserved on redirect)

Reply:
I think that this will not interfere with the main logic. The code(type) of redirect is important